network-api/networkapi/wagtailpages/models.py

```diff
@@ -52,6 +52,7 @@
     ('pulse_listing', customblocks.PulseProjectList()),
     ('profile_listing', customblocks.LatestProfileList()),
     ('profile_by_id', customblocks.ProfileById()),
+    ('profile_directory', customblocks.ProfileDirectory()),
```

Comment:
Since we'll be landing this in parts, let's lock this behind a feature flag:
```
base_fields = base_fields = [field for field in [
    ...
    ('profile_by_id', customblocks.ProfileById()),
    ('profile_directory', customblocks.ProfileDirectory()) if settings.ENABLE_PROFILE_DIRECTORY_BLOCK else None,
    ('airtable', customblocks.AirTableBlock()),
] if field is not None]
```
which requires a small update to `settings.py`. Add this in the `env` block at the top:
```
    ...
    DOMAIN_REDIRECT_MIDDLWARE_ENABLED=(bool, False),
    ENABLE_PROFILE_DIRECTORY_BLOCK =(bool, False),
    FILEBROWSER_DEBUG=(bool, False),
    ...
```
and then below the INSTALLED_APPS section, add this:
```
# Wagtail feature flags
ENABLE_PROFILE_DIRECTORY_BLOCK = env('ENABLE_PROFILE_DIRECTORY_BLOCK ')
```
That way, we can land this code, and even promote it to production, without it actually doing anything until we're either ready to flip that switch, or we decide to do a final code update to remove the feature flag condition.